package.json

```diff
@@ -12,6 +12,9 @@
   "homepage": "https://github.com/bampakoa/ngx-electronify#readme",
   "main": "dist/index.js",
   "builders": "builders.json",
+  "ng-add": {
+    "save": "dependencies"
```

Comment:
Does this mean that essentially that `ng add` will also install it as a dependency?

Reply:
Yes it does, let me show you the part in the `angular-cli` that does this.